log4j-core/src/test/java/org/apache/logging/log4j/core/appender/TlsSyslogAppenderTest.java

```diff
@@ -94,7 +94,7 @@ private SyslogAppender createAppender() {
         }
 
         return SyslogAppender.createAppender("localhost", PORTNUM, "SSL", sslConfiguration, 0, -1, true, "Test", true,
-            false, Facility.LOCAL0, "Audit", 18060, true, "RequestContext", null, null, includeNewLine, null,
+            false, Facility.LOCAL0, "Audit", "18060", true, "RequestContext", null, null, includeNewLine, null,
```

Comment:
No need to modify the tests if backward compatibility is maintained.

Reply:
You mean - existing tests should not be modified. New ones should be created to test the new functionality.